src/hermes/commands/deposit/invenio.py

```diff
@@ -205,7 +218,12 @@ def _resolve_latest_invenio_id(ctx: CodeMetaContext) -> str:
         # If we got a record id by now, resolve it using the Invenio API to the latests record.
         return _invenio_resolve_record_id(site_url, record_id)
 
-    raise ValueError("Could not figure out how to retrieve record id.")
+    # TODO figure out whether an initial deposit is what we want...
+    trigger_initial_deposit = False
+    if trigger_initial_deposit:
+        return None, {}
+    else:
+        raise ValueError("Could not figure out how to retrieve record id.")
```

Comment:
Whether we want to trigger an initial deposit should not influence the result of this function. I would always return the same thing (`None, {}` is fine, I think) and move this decision elsewhere.

Reply:
Yeah, totally agree. Even more as this is obviously bloat code